src/ocaml_specific.ml

```diff
@@ -456,7 +457,36 @@ rule "ocamldoc: document ocaml project odocl & *odoc -> latex|dot..."
         The target foo.docdir/bar.dot will generate a dot graph of module \
         dependencies."
   (Ocaml_tools.document_ocaml_project
-      ~ocamldoc:Ocaml_tools.ocamldoc_l_file "%(dir).odocl" "%(dir).docdir/%(file)" "%(dir).docdir");;
+      ~tags:["texdoc"]
+      ~ocamldoc:Ocaml_tools.ocamldoc_l_file
+        "%(dir).odocl" "%(dir).docdir/%(file).tex" "%(dir).docdir");;
+
+rule "ocamldoc: document ocaml project odocl & *odoc -> latex (.ltx)"
+  ~prod:"%(dir).docdir/%(file).ltx"
+  ~dep:"%(dir).odocl"
+  ?doc:None (* TODO document *)
```

Comment:
I'm not sure whether this is a remnant from a patch developed before I merged #20, or whether it is because you added new rules, but in any case you must fix this :-)